src/tests/ftest/util/server_utils.py

```diff
@@ -516,26 +530,48 @@ def clean_files(self):
         clean_cmds = [
             "find /mnt/daos -mindepth 1 -maxdepth 1 -print0 | xargs -0r rm -rf"
         ]
-
-        if self.runner.job.yaml_params.is_nvme() or \
-           self.runner.job.yaml_params.is_scm():
+        if self.runner.job.yaml_params.is_nvme():
             clean_cmds.append("sudo rm -rf /mnt/daos; sudo umount /mnt/daos")
-
+        # scm_mount can be /mnt/daos0 or /mnt/daos1 for two daos_server
+        # instances. Presently, not supported in DAOS. The for loop needs
+        # to be updated in future to handle it. Single instance pmem
+        # device should work now.
+        if self.runner.job.yaml_params.is_scm():
+            scm_mount_count = len(self.runner.job.yaml_params.
+                                  server_params[-1].scm_list.value)
+            for count in range(0, scm_mount_count):
+                clean_cmds.append("sudo umount /mnt/daos; \
+                                   sudo wipefs -a /dev/pmem{}".format(count))
```

Comment:
Shouldn't  this be using the scm mount point name that is defined in the daos server yaml file, e.g.
```
            from ast import literal_eval
            ...
            for server_params in self.runner.job.yaml_params.server_params:
                # Convert the list represented as a string into a list
                for scm_mount in literal_eval(server_params.scm_list.value):
                    # Add a wipefs command for each scm mount point specified in the yaml
                    clean_cmds.append("sudo wipefs -a {}".format(scm_mount))
```

Reply:
Good point. Will do...

Reply:
Done.